frame/contracts/src/lib.rs

```diff
@@ -499,6 +518,7 @@ pub mod pallet {
 			storage_deposit_limit: Option<<BalanceOf<T> as codec::HasCompact>::Type>,
 			determinism: Determinism,
 		) -> DispatchResult {
+			Migration::<T>::ensure_migrated()?;
```

Comment:
mm I think it's fine here, and since we repeat that a couple of time it's nice to have a slightly shorter variant. we used the same word `ensure` so frame devs should not be too surprised about the behaviors.